### PR TITLE
Adding a path to JEP for MacOs

### PR DESCRIPTION
**Pull request by akuleshov7**
### What's done:
- To build a python frontend on Mac you need to provide a path to a proper brew installation on MacOs

### CODE REVIEW

File: cpg-language-python/src/main/kotlin/de/fraunhofer/aisec/cpg/frontends/python/JepSingleton.kt [hunk 1]
```diff
@@ -112,47 +120,42 @@ object JepSingleton {
...
     }
+
+    private fun libJepSoFromVirtualEnv(version: String) =
+        "${System.getProperty("user.home")}/.virtualenvs/$virtualEnv/lib/python${version}/site-packages/jep/$libJepSo"
+
+    private fun libJepJniFromVirtualEnv(version: String) =
+        "${System.getProperty("user.home")}/.virtualenvs/$virtualEnv/lib/python${version}/site-packages/jep/$libJepJniLib"
+
+    private fun macOsInstallationPath(version: String) =
+        "/opt/homebrew/lib/python${version}/site-packages/jep/$libJepJniLib"
```
Comment (akuleshov7) [inline cpg-language-python/src/main/kotlin/de/fraunhofer/aisec/cpg/frontends/python/JepSingleton.kt:154]: without this path - there will be a problem to run tests locally on latest Mac with brew installations of python and jep

**Comment by akuleshov7** [comment]
@maximiliankaul is this repository alive or archived?

**Comment by maximiliankaul** [comment]
Hi akuleshov7,
thanks for you contribution. Unfortunately, we cannot merge this PR right now until our legal team has implemented a CLA for external contributions. We are unhappy with this situation, too. I have clarified this in the readme and hope this blocker will be resolved soon.

**Comment by akuleshov7** [comment]
> > > Hi akuleshov7,
> > > thanks for you contribution. Unfortunately, we cannot merge this PR right now until our legal team has implemented a CLA for external contributions. We are unhappy with this situation, too. I have clarified this in the readme and hope this blocker will be resolved soon.
> > 
> > 
> > wtf, you are under an apache 2.0 license, you have plans to change it?
> 
> First of all, again thank you for your contribution. Just a little bit of explanation. Your PR was among the first ones that are coming from external. So far we only had contributions from internal employees. Because we are part of the overall Fraunhofer-Gesellschaft e.V. we are obliged to follow some of the central guidelines when it comes to "legal" issues, also obliged to keeping the non-profit status. While we have no immediate plans to change the license (in fact we are quite happy with Apache 2.0 for this project), we have the requirement of establishing an CLA once external contributions come into play.

I love to be the first contributor 🥰 and I am also thinking that you are also very happy to see external people getting into your project 😄  (at least you can now report to your management that making the project open-sourced was not a mistake). But that is very confusing for me as I see such problem for a first time in my life in open source 🤣

**Comment by oxisto** [comment]
> > > > Hi akuleshov7,
> > > > thanks for you contribution. Unfortunately, we cannot merge this PR right now until our legal team has implemented a CLA for external contributions. We are unhappy with this situation, too. I have clarified this in the readme and hope this blocker will be resolved soon.
> > > 
> > > 
> > > wtf, you are under an apache 2.0 license, you have plans to change it?
> > 
> > 
> > First of all, again thank you for your contribution. Just a little bit of explanation. Your PR was among the first ones that are coming from external. So far we only had contributions from internal employees. Because we are part of the overall Fraunhofer-Gesellschaft e.V. we are obliged to follow some of the central guidelines when it comes to "legal" issues, also obliged to keeping the non-profit status. While we have no immediate plans to change the license (in fact we are quite happy with Apache 2.0 for this project), we have the requirement of establishing an CLA once external contributions come into play.
> 
> I love to be in the list of first contributors 🥰 and I am also thinking that you are also very happy to see external people getting into your project 😄 (at least you can now report to your management that making the project open-sourced was not a mistake). But that is very confusing for me as I see such problem for a first time in my long life in open source 🤣

Yeah I feel you :) Unfortunately we dragged this issue along for quite a while and now we can/will solve it :)

**Comment by akuleshov7** [comment]
Closed in a favour of https://github.com/Fraunhofer-AISEC/cpg/pull/1400.

**Pull request closed without merging**